### ISSUE TITLE
MapML

### ISSUE THREAD

**Issue by tidoust** [w3c/strategy#115]
The [Maps for HTML Community Group](https://www.w3.org/community/maps4html/), driven by Peter Rushforth (@prushforth) develops proposals to define a new `<map>` element for HTML to represent a map.

There are two proposals in one:
- [The HTML `<map>` Element proposal](https://maps4html.github.io/HTML-Map-Element/spec/) which extends the semantics of the `<map>` element to introduce map-related features such as the notion of layers and of projections.
- [Map Markup Language](https://maps4html.github.io/MapML/spec/) (MapML) which is a text format for encoding map information on the Web

Also see the [minutes of MapML discussions](https://www.w3.org/2018/02/20-sdw-minutes.html#x14) during the Spatial Data on the Web IG F2F in Amersfoort, 20 February 2018

**Comment by prushforth** [comment]
For those who are interested, although the document is a bit old now, I set out some [use cases and requirements for declarative maps on the Web](http://maps4html.github.io/HTML-Map-Element-UseCases-Requirements/) when I first started working on this, they may still be relevant however and they might answer some questions that I noticed from our face to face meeting that may still be outstanding.  I am working on a spreadsheet that relates MapML and Maps for HTML goals to the excellent [Spatial Data on the Web Best Practices](https://www.w3.org/TR/sdw-bp/).  As such it will take me a bit of time as there are many points in common.

**Comment by prushforth** [comment]
Please have a look at the [attached spreadsheet](https://github.com/w3c/strategy/files/1820342/MapML.Value.Proposition.xlsx), including comments.  I have tried to give a sense of "what MapML aims to achieve / how is this different from where we are now", which were questions asked of me during our [face to face meeting](https://www.w3.org/2018/02/20-sdw-minutes.html#x14) and re-iterated at [this week's monthly meeting](https://www.w3.org/2018/03/14-sdw-minutes.html#x01). 

In particular, it was mentioned that: 

> our concern is to understand how MapML is better than what we have today, so that we may sell it

so I have tried to enumerate how MapML, and of course Maps for HTML seeks to change the status quo, in some cases by highlighting how the objectives embody (some of) the best practices published by the Spatial Data on the Web working group, and the related Data on the Web working group, and other relevant documents.  

Hopefully this and the above mentioned use cased / requirements document will give rise to some fruitful discussion, here or on the [Maps for HTML repos](https://github.com/Maps4HTML).

**Comment by 6a6d74** [comment]
Note that this discussion is also paralleled over in the w3c/sdw repo, [Issue 1024](https://github.com/w3c/sdw/issues/1024)

**Comment by prushforth** [comment]
Yes, I mistakenly commented here, and I was concerned that SDWIG members would not notice this thread so I copied my comment over there, which is likely the more appropriate venue to discuss that particular issue.

**Comment by 6a6d74** [comment]
@prushforth 

During the SDW IG Face to Face meeting last month, you said that you hoped to organise a MapML Community Group meeting for TPAC 2018 during the Wednesday plenary session.

Is there any progress on this?

**Comment by 6a6d74** [comment]
(on behalf of @prushforth)

[Peter has] opened an [issue on the Testbed 14 MapML ER](https://github.com/opengeospatial/D012-MapML_Engineering_Report/issues/23) to invite collaborative discussion about the road to standardization.  Your comments here or there are welcome.

**Comment by prushforth** [comment]
I have put a [proposal](https://www.w3.org/wiki/TPAC/2018/SessionIdeas#Session:_Maps_for_HTML) on the TPAC breakout session wiki. My plan is to create a wiki page and add some more substance to the proposal. I gather that there is a mad scramble to get sessions approved, and that they give a bit more leeway to first-time TPAC participants, so hopefully it will get enough support to get accepted as a session.  It's not clear to me how sessions get voted on or accepted.

**Comment by tidoust** [comment]
>  It's not clear to me how sessions get voted on or accepted.

There are no votes or acceptation procedure. I don't think we've ever run in a situation where we had to discard proposals, although this could happen in theory. The exact schedule gets decided on a white boards through post-its on Wednesday morning. From time to time, some sessions get merged because they talk about related topics.

Now, this doesn't guarantee that the breakout session will attract a fair number of participants, so it's always good to reach out to people you'd like to see attend before the event, and live before the breakout session. It's also important to avoid scheduling the session at the same time as another potentially conflicting session.

**Comment by prushforth** [comment]
We are planning and hoping to hold a [W3C Workshop on Maps for the Web](https://github.com/w3c/strategy/issues/210), and I'll be presenting our [MapML proposal ](https://github.com/Maps4HTML/MapML-Proposal#the-mapml-proposal---explainermd) for discussion there, assuming we get the go-ahead.

**Comment by prushforth** [comment]
Apologies for the late notice; lot's of water under the bridge since the last comment.  Thanks @6a6d74 for pinging me to update these links.  

The [Joint W3C / OGC Workshop on Maps for the Web](https://www.w3.org/2020/maps/) will be held online, Sept 21st - Oct 2nd 2020.  The CFP closes June 30th.  Still looking for more 'positions', and potentially a few more presentations, but it is shaping up quite nicely now.

**Comment by prushforth** [comment]
The [report from the joint W3C / OGC workshop on Maps for the Web is available here](https://www.w3.org/2020/maps/report).

**Comment by prushforth** [comment]
A [report on activity by the Maps for HTML Community Group since, and motivated by the workshop, is available here](https://www.w3.org/community/maps4html/2021/05/27/winter-2021-report/)
